pandas/tests/io/test_parquet.py

```diff
@@ -201,7 +201,8 @@ def test_options_get_engine(fp, pa):
 
 
 @pytest.mark.xfail(is_platform_windows() or is_platform_mac(),
```

Comment:
Can you try removing the xfail entirely. It looks like it may be unnecessary now.

Reply:
I did do that in another branch and it passed the CI tests, but it seems there's discussion of pyarrow and fastparquet as well as windows causing problems? Not completely sure.